CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.3)
 
-project(rabit VERSION 0.2.0)
+project(rabit VERSION 0.2.1)
 
 option(RABIT_BUILD_TESTS "Build rabit tests" OFF)
 option(RABIT_BUILD_MPI "Build MPI" OFF)
-option(RABIT_BUILD_DMLC "Include DMLC_CORE in build" ON)
+option(RABIT_BUILD_DMLC "Include DMLC_CORE in build" OFF)
```

Comment:
why this PR involves changes on make/cmake files?

Reply:
this is a bug fix, set on means we expect dmlc-core subdirectory in rabit. It is the case for travis ci as we do run script and download project. In xgboost project, rabit by default should use dmlc-core subproject in same directory level as rabit.

Reply:
but the build in xgboost is always succeeded?

Reply:
reverted